.babelignore

```diff
@@ -1,3 +1,4 @@
 # Ensure babel-register won't compile fixtures, or try to recompile compiled code.
 packages/*/test/fixtures
 packages/*/lib
+packages/babel-standalone/babel.js
```

Comment:
do we need to add min.js too?

Reply:
We could, but we never `require()` that file, so it's probably not worth worrying about.

I just noticed it because we were getting the `deoptimize` warning when running the tests:
```
$ make test
./node_modules/.bin/eslint scripts packages *.js --format=codeframe --rulesdir="./eslint_rules"
./scripts/test.sh
[BABEL] Note: The code generator has deoptimised the styling of "/Users/thesmyth/projects/babel/packages/babel-standalone/babel.js" as it exceeds the max of "500KB".


  ․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․
  ․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․
```